git_pile/pile.py

```diff
@@ -235,19 +232,19 @@ def text(self, *path):
         return self.__path.joinpath(*path).read_text()
 
     def sha1(self, *path):
-        return _git(["hash-object", "--", str(self.__path.joinpath(*path))]).stdout.strip()
```

Comment:
same as above, although we should probably have just one way for git().... leaving it as helpers.git() in some places and as git() in others IMO is not good

Reply:
Yeah... So I'm more inclined to refer to module contents using the `<module>.<name>` syntax, since it is clear to me that `<name>` comes from `<module>`. But I agree that inconsistency is not good.

I'll update this to import git from helpers.